tests/snmp/test_snmp_loopback.py

```diff
@@ -25,9 +27,15 @@ def get_snmp_output(ip, duthost, nbr, creds_all_duts):
         iptables_cmd, ip)
     duthost.shell(ip_tbl_rule_add)
 
-    eos_snmpget = "bash snmpget -v2c -c {} {} 1.3.6.1.2.1.1.1.0".format(
-        quote(creds_all_duts[duthost.hostname]['snmp_rocommunity']), ip)
-    out = nbr['host'].eos_command(commands=[eos_snmpget])
+    if isinstance(nbr['host'], EosHost):
```

Comment:
I have another sonic-mgmt PR which will handle this change https://github.com/sonic-net/sonic-mgmt/pull/8802 
You could remove these changes from test_snmp_loopback.py

Reply:
Fixed